Sources/FioriSwiftUICore/_ComponentProtocols/CompositeComponentProtocols.swift

```diff
@@ -467,11 +467,15 @@ protocol _RatingControlFormViewComponent: _TitleComponent, _RatingControlCompone
 }
 
 // sourcery: CompositeComponent
-protocol _ProfileHeaderComponent: _DetailImageComponent, _TitleComponent, _SubtitleComponent, _DescriptionComponent {
+protocol _ProfileHeaderComponent: _DetailImageComponent, _TitleComponent, _SubtitleComponent, _DescriptionComponent, _SeparatorComponent {
     // sourcery: defaultValue = "false"
     /// Indicate whether the profile header was animatable in scroll view. The default was false.
     var animatable: Bool { get }
     
+    // sourcery: defaultValue = false
+    /// Provides a standard hairline for profile header or not. The default value is `false`.
```

Comment:
```suggestion
    /// Provides a standard hairline for profile header or not. The default value is `true`.
```
Are we supposed to make this as true, as per discussion with Designers?